### PR TITLE
[WA-37]: created notification for a limit of favorites

### PR DESCRIPTION
**Pull request by uselesswhale1**
- created notification for a limit of favorites
- added notification on heart icon (so notification service can be shown)
- notification service is now ready to use

### CODE REVIEW

File: src/app/shared/components/favorite-toggle/favorite-toggle.component.ts [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-  constructor() {}
+  constructor(private notificationService: NotificationService) {}
   @Input() isFavorite: boolean | undefined;
   @Output() onChange = new EventEmitter<boolean | undefined>();
 
   onToggle() {
     this.isFavorite = !this.isFavorite;
     this.onChange.emit(this.isFavorite);
+
+    this.notificationService.show('Info','You clicked on heart icon')
```
Comment (dmitry-chirva) [inline src/app/shared/components/favorite-toggle/favorite-toggle.component.ts:18]: Please create Enum for type of notifications

File: src/core/api/notification/notification.service.ts [hunk 2]
```diff
@@ -16,6 +16,10 @@ export class NotificationService {
     this.notifications.push({id, type, message});
```
Comment (dmitry-chirva) [inline src/core/api/notification/notification.service.ts:16]: ?? why do we need that ?

File: src/core/favorite/favorite.service.ts [hunk 3]
```diff
@@ -46,12 +49,70 @@ export class FavoriteService {
...
+    // this.getFavorites().subscribe((v)=>{console.log(v)});
+
+    this.getFavorite().subscribe(value=> {
+      console.log(value);
+    });
+  }
+
+
+
+  getFavorite(): Observable<CityWeatherInfo>{
```
Comment (dmitry-chirva) [inline src/core/favorite/favorite.service.ts:65]: Communicate with Kate and Oleksii, this is old implementation or I will wait their implementation

File: src/app/shared/interfaces/forecast-info.interfaces.ts [hunk 4]
```diff
@@ -51,22 +52,22 @@ export interface Forecastday {
...
-  totalprecip_mm: number;
-  avgvis_km: number;
-  avghumidity: number;
-  daily_will_it_rain: number;
-  daily_chance_of_rain: number;
-  daily_will_it_snow: number;
-  daily_chance_of_snow: number;
-  condition: Condition2;
-}
+// export interface Day {
```
Comment (dmitry-chirva) [inline src/app/shared/interfaces/forecast-info.interfaces.ts:55]: Remove or uncomment interface

**Comment by uselesswhale1** [comment]
Did comment @dmitry-chirva wrote:
- merged current version from 'develop'
- removed comments (merged current version of forecast-info.interfaces.ts)
- returned old way of getting fav-items in fav-list (new way is in progress)

File: src/app/shared/components/favorite-toggle/favorite-toggle.component.ts (hunk 1 above)
Comment (uselesswhale1) [reply src/app/shared/components/favorite-toggle/favorite-toggle.component.ts]: Created Enums for type of notifications

File: src/core/api/notification/notification.service.ts (hunk 2 above)
Comment (uselesswhale1) [reply src/core/api/notification/notification.service.ts:16]: All was right, no fixes needed

**Pull request merged**